From 3773f79162835a9b6fa7f7359ff6f4ed579bbf4a Mon Sep 17 00:00:00 2001
From: Max Ihlenfeldt <max@igalia.com>
Date: Mon, 7 Aug 2023 13:26:55 +0000
Subject: [PATCH] Backport "IWYU for chromedriver/chrome/client_hints.h"

This is a backport of a patch that fixes a build error due to a missing
import. It will be included in upstream's 118 release.

Upstream-Status: Backport [https://crrev.com/c/4767547]
Signed-off-by: Max Ihlenfeldt <max@igalia.com>
---
 chrome/test/chromedriver/chrome/client_hints.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/chrome/test/chromedriver/chrome/client_hints.h b/chrome/test/chromedriver/chrome/client_hints.h
index e737376..42dc1ac 100644
--- a/chrome/test/chromedriver/chrome/client_hints.h
+++ b/chrome/test/chromedriver/chrome/client_hints.h
@@ -7,6 +7,7 @@

 #include <third_party/abseil-cpp/absl/types/optional.h>
 #include <string>
+#include <vector>

 struct BrandVersion {
   // Browser brand.
